torch/csrc/lazy/core/dynamic_ir.h

```diff
@@ -49,6 +49,7 @@ class TORCH_API DimensionNode {
   virtual int64_t getStaticValue() const {
     TORCH_CHECK(false, "NYI");
   };
+  virtual ~DimensionNode() {};
```

Comment:
I think `virtual ~DimensionNode() = default;` would be better practice here. Same for below.